code/game/objects/items/scanners.dm

```diff
@@ -107,7 +107,7 @@ REAGENT SCANNER
 	if(isxeno(M))
 		balloon_alert(user, "Unknown entity")
 		return
-	if(M.species.species_flags & NO_SCAN)
+	if(M.species.species_flags & IS_SYNTHETIC)
```

Comment:
?? the no_scan flag is specifically to... make you not scannable.

Reply:
Yeah it was a hack to not remove the no_scan flag from robots. fixed